bundle/src/main/java/com/adobe/acs/commons/remoteassets/impl/RemoteAssetsConfigImpl.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.remoteassets.impl;
+
+import com.adobe.acs.commons.remoteassets.RemoteAssetsConfig;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.felix.scr.annotations.Activate;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.ConfigurationPolicy;
+import org.apache.felix.scr.annotations.Deactivate;
+import org.apache.felix.scr.annotations.Modified;
+import org.apache.felix.scr.annotations.Property;
+import org.apache.felix.scr.annotations.Service;
+import org.apache.sling.commons.osgi.PropertiesUtil;
+import org.osgi.service.component.ComponentContext;
+
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Configuration service for Remote Asset feature. Implements {@link RemoteAssetsConfig}.
+ */
+@Component(
+        immediate = true,
+        metatype = true,
+        label = "ACS AEM Commons - Remote Assets - Config",
+        policy = ConfigurationPolicy.REQUIRE
+)
+@Service()
+public class RemoteAssetsConfigImpl implements RemoteAssetsConfig {
+
+    @Property(label = "Server")
+    private static final String SERVER = "server";
+
+    @Property(label = "Username")
+    private static final String USERNAME = "username";
+
+    @Property(label = "Password")
+    private static final String PASSWORD = "password";
+
+    @Property(
+            label = "Tag Sync Paths",
+            description = "Paths to sync tags from the remote server (e.g. /etc/tags/{asset})",
```

Comment:
because we are saying this is an example, I wouldnt put `{asset}` in there b/c that would not actually be a valid example.  I think somehting like `/etc/tags/assets` is fine.  Not sure with /content/dam below but I can see either /content/dam for the entire (since it may be valid, whereas you'd probably not want to intermingle all of /etc/tags) or /content/dam/myfolder